README.asciidoc

```diff
@@ -0,0 +1,89 @@
+= This is the documentation file for team TWELVE.
+
+=== Team members and their main roles are:
+
+* Pille Pärnalaas - Programming
+* Kenno Kallastu - Mechanics
+* Stepan Denysenko - Electronics
+
+
+== DAILY BLOG
+
+=== 03.09
+====== BOOTCAMP DAY ONE
+Kenno, Pille:: present and working the tasks (both 8h)
+
+=== 14.09
+First gathering of the team. Divided main roles and started work.
+Kenno:: Omnidirectional wheels attached to the chassis along with the motors. Helped with electronics. 
+	Getting familiar with the lab.(1.5h)
+Stepan:: Wired, soldered. (1.5h)
+Pille:: Took home the NUC for initial setup. (1h)
```

Comment:
Add blank line before description list (`<text>::`), otherwise the syntax does not work.